src/aggregator/server/rawtcp/server_test.go

```diff
@@ -249,42 +221,34 @@ func testRawTCPServerHandleUnaggregated(
 			require.NoError(t, err)
 
 			var stream []byte
-			switch protocol {
-			case msgpackEncoding:
-				encoder := msgpack.NewUnaggregatedEncoder(msgpack.NewPooledBufferedEncoder(nil))
-				require.NoError(t, encoder.EncodeCounterWithPoliciesList(testCounterWithPoliciesList))
-				require.NoError(t, encoder.EncodeBatchTimerWithPoliciesList(testBatchTimerWithPoliciesList))
-				require.NoError(t, encoder.EncodeGaugeWithPoliciesList(testGaugeWithPoliciesList))
-				stream = encoder.Encoder().Bytes()
-			case protobufEncoding:
-				encoder := protobuf.NewUnaggregatedEncoder(protobuf.NewUnaggregatedOptions())
-				require.NoError(t, encoder.EncodeMessage(encoding.UnaggregatedMessageUnion{
-					Type:                 encoding.CounterWithMetadatasType,
-					CounterWithMetadatas: testCounterWithMetadatas,
-				}))
-				require.NoError(t, encoder.EncodeMessage(encoding.UnaggregatedMessageUnion{
-					Type: encoding.BatchTimerWithMetadatasType,
-					BatchTimerWithMetadatas: testBatchTimerWithMetadatas,
-				}))
-				require.NoError(t, encoder.EncodeMessage(encoding.UnaggregatedMessageUnion{
-					Type:               encoding.GaugeWithMetadatasType,
-					GaugeWithMetadatas: testGaugeWithMetadatas,
-				}))
-				require.NoError(t, encoder.EncodeMessage(encoding.UnaggregatedMessageUnion{
-					Type: encoding.TimedMetricWithMetadataType,
-					TimedMetricWithMetadata: testTimedMetricWithMetadata,
-				}))
-				require.NoError(t, encoder.EncodeMessage(encoding.UnaggregatedMessageUnion{
-					Type: encoding.PassthroughMetricWithMetadataType,
-					PassthroughMetricWithMetadata: testPassthroughMetricWithMetadata,
-				}))
-				require.NoError(t, encoder.EncodeMessage(encoding.UnaggregatedMessageUnion{
-					Type: encoding.ForwardedMetricWithMetadataType,
-					ForwardedMetricWithMetadata: testForwardedMetricWithMetadata,
-				}))
-				buf := encoder.Relinquish()
-				stream = buf.Bytes()
-			}
+			encoder := protobuf.NewUnaggregatedEncoder(protobuf.NewUnaggregatedOptions())
```

Comment:
nit: after these changes, `stream` doesn't need to be declared until below when it's actually assigned